bessctl/commands.py

```diff
@@ -1334,6 +1358,33 @@ def tcpdump_module(cli, module_name, ogate, opts):
         except:
             pass
 
+@cmd('track MODULE [DIRECTION] [GATE] [PRIORITY]',
```

Comment:
Ah, this is cleaner than what I had in mind for the global enable/disable case. Will do.